.circleci/config.yml

```diff
@@ -48,7 +48,16 @@ jobs:
             - ./node_modules
       - run: npm run build
       - run: npm run test
-
+  test-environment:
+    docker:
+      - image: cimg/base:2021.10
```

Comment:
I didn't use the ubuntu image ([as suggested](https://github.com/mozilla/telemetry-airflow/blob/140d900b85c2a261f025631313fc93afe08d6055/.circleci/config.yml#L47-L49)) because it gave this error:

```
Error: 
Error response from daemon: pull access denied for ubuntu-1604, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
```

Reply:
Yeah I think in this case using a more recent/common docker image makes sense. I don't think it really matters a ton as long as it has docker / docker-compose installed.